status/projects.py

```diff
@@ -696,16 +696,18 @@ class RunningNotesDataHandler(SafeHandler):
     """
     def get(self, project):
         self.set_header("Content-type", "application/json")
-        p = Project(lims, id=project)
+        v = self.application.projects_db.view("project/project_id")
+        for row in v[project]:
+            doc_id = row.value
         try:
-            p.get(force=True)
+            doc = self.application.projects_db.get(doc_id)
```

Comment:
will this raise some strange 'variable not defined' exception when the project is not found? It feels a bit cleaner to have the check a bit earlier? Alternatively if you specify the exact exception that will be raised, it is easier to understand if this is how you want it.